jest/setup.js

```diff
@@ -8,6 +8,8 @@
  */
 'use strict';
 
+var immutable = require('immutable');
```

Comment:
immutable shouldn't require here, it should happen inside of the `jest.mock` function for this.